src/components/squareComponent/squareComponent.js

```diff
@@ -2,15 +2,30 @@ import React, { Component } from 'react';
 import './squareComp.css';
 
 class Square extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      valueX: "",
```

Comment:
What does `valueX` and `valueO` represent?

Reply:
`valueX` and `valueO` represent `X` and `O` respectively.  I can rename the variables as `X` and `O` to enhance readability.

Reply:
I'm still not sure I understand. Why are they being tracked separately?

Reply:
This way I can track when it is either X or O's turn.  In order to track if X or O has gone I decided to track them individually.

Reply:
Let's chat about this tomorrow.